<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_popSize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Population size</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_noGen">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>250</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Number of generations</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdit_PopSize">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtEdit_NoGen">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>100</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Start">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fitness">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Clear">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="CuttingProblem.qrc"/>
 </resources>
 <connections/>
</ui>
